docs/source/online_dpo_trainer.md

```diff
@@ -7,6 +7,41 @@ While [Guo et al. (2024)](https://huggingface.co/papers/2402.04792) used a LLM j
 
 ## Get started
 
+The basic API looks as follows:
+
+```python
+from datasets import Dataset
+from trl import OnlineDPOConfig, OnlineDPOTrainer
+from transformers import (
+    AutoModelForCausalLM,
+    AutoModelForSequenceClassification,
+    AutoTokenizer,
+)
+NUM_DUMMY_SAMPLES = 100
+tok = AutoTokenizer.from_pretrained("gpt2")
+tok.add_special_tokens({"pad_token": "[PAD]"})
```

Comment:
Is this a quirk of GPT2? Is so, WDYT about using a different model like one of the Qwen2 or SmolLM ones?

Reply:
Let's use SmolLM